check_opnsense.py

```diff
@@ -273,6 +310,144 @@ def check_ipsec(self) -> None:
         self.perfdata.append(f"tunnels_connected={len(tunnels_connected)}")
         self.perfdata.append(f"tunnels_disconnected={len(tunnels_disconnected)}")
 
+    def check_hastatus(self) -> None:
```

Comment:
As long this is WIP / not implemented, please move this feature into a separate (draft) PR